### PR TITLE
Feature/bearc sbnd supera fcls

### PR DESCRIPTION
**Pull request by bear-is-asleep**
Supera fcls for overlay and mpvmpr samples. Dependent on some PRs in `sbncode` and `LArG4`
* https://github.com/LArSoft/larg4/pull/52
* https://github.com/SBNSoftware/sbncode/pull/404

### CODE REVIEW

**Comment by bear-is-asleep** [comment]
gotta track the ancestory down to label the truth interactions properly

**Comment by yeonjaej** [comment]
I don't have the merge access. the PR looks good.

**Comment by bear-is-asleep** [comment]
Just updated so the fcls work for the grid. Basically the art root output needs to have a unique name when making file lists, which is handled by sbnd's root output. Unfortunately, `sbndcode` is needed to run these fcls, but this has always been the case. It doesn't make sense to move it to `sbndcode` since all of the modules to run it exist here. Can you check/approve these when you get a chance @francois-drielsma ? Thanks!

**Review by francois-drielsma — APPROVED** [review]

**Pull request merged**
